### PR TITLE
fixing contributor list

### LINKED ISSUE #492: Update contributor list
I.e. here
https://github.com/pytrip/pytripgui/blob/master/pytripgui/messages.py

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #492

### CODE REVIEW

File: pytripgui/messages.py [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-about_txt_en += "Previous contributors:\n"
-about_txt_en += "    Jakob Toftegaard\n"
+about_txt_en = """PyTRipGUI Version: {}
+PyTRiP Version: {}
+(c) 2010 - 2021 PyTRiP98 Developers
+    Niels Bassler
+    Leszek Grzanka
+    Toke Printz
+    Łukasz Jeleń
+    Arkadiusz Cwikła
```
Comment (rozchlastywacz) [inline pytripgui/messages.py:11]: if you are using polish letters, please use them in mine name as well :)

File: setup.py [hunk 2]
```diff
@@ -39,8 +39,10 @@ def write_version_py(filename=os.path.join('pytripgui', 'VERSION')):
     packages=setuptools.find_packages(exclude="tests"),
     url='https://github.com/pytrip/pytripgui',
     license='GPL',
-    author='Jakob Toftegaard, Niels Bassler, Leszek Grzanka',
-    author_email='leszek.grzanka@gmail.com',
+    author='Niels Bassler at el.',
```
Comment (rozchlastywacz) [inline setup.py:42]: did you mean `et al`?

**Review by rozchlastywacz — CHANGES_REQUESTED** [review]

File: pytripgui/messages.py [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
+    Joanna Fortuna
+    Michal Krawczyk
```
Comment (rozchlastywacz) [inline pytripgui/messages.py:13]: and also in his name ;)

File: setup.py [hunk 4]
```diff
@@ -39,8 +39,10 @@ def write_version_py(filename=os.path.join('pytripgui', 'VERSION')):
     packages=setuptools.find_packages(exclude="tests"),
     url='https://github.com/pytrip/pytripgui',
     license='GPL',
-    author='Jakob Toftegaard, Niels Bassler, Leszek Grzanka',
-    author_email='leszek.grzanka@gmail.com',
+    author='Niels Bassler et el.',
```
Comment (michal367) [inline setup.py:42]: et al.

Comment (grzanka) [reply setup.py]: done

**Review by michal367 — APPROVED** [review]

**Review by rozchlastywacz — APPROVED** [review]
Nice

**Pull request merged**